inspirehep/dojson/utils/__init__.py

```diff
@@ -224,3 +227,8 @@ def dedupe_all_lists(obj):
         return type(obj)(new_obj)
     else:
         return obj
+
+
+def validate(instance, schema):
+    return jsonschema_validate(
+        instance, schema, resolver=LocalRefResolver('', {}))
```

Comment:
The plan is to write as many tests as possible for HEP and HEPNames that also validate the produced schema. These make sense already _now_, so I want to tackle one problem at a time.